packages/ui-pagination/src/Pagination/index.tsx

```diff
@@ -104,7 +104,7 @@ class Pagination extends Component<PaginationProps> {
     currentPage: 1,
     siblingCount: 1,
     boundaryCount: 1,
-    ellipsis: <li style={{all: 'unset'}}>...</li>,
```

Comment:
I think that this was a bug too, that its a `li` should be an internal part of the component